packages/core/src/components/search/Filter/FilterDesktop.tsx

```diff
@@ -73,13 +75,8 @@ function FilterDesktop({
     toggleFacet: onFacetChange,
     fallbackToFirst: true,
     allFacets: facets,
-    deliverySettings,
-  })
-
-  const regionalizationData = getRegionalizationSettings({
-    deliverySettings,
+    deliveryPromiseSettings,
   })
-  const { deliverySettings: deliverySettingsData } = regionalizationData
```

Comment:
Not necessary to send deliverySettings, because the only info that can come from a section is related to regionalization (inputField or idkPostalCodeLink) -- the idea of getGlobalSettings is to receive the global settings and, if there were already some section settings, getGlobalSettings merges them (global and section). The merge only happens for RegionModal and RegionPopover because they're the sections that still have those deprecated settings.